### PR TITLE
{bio}[dummy/] IGV 2.4.6 Java-1.8.0_152 (WIP)

### PR DESCRIPTION
**Pull request by nathanhaigh**
As with #5668, given that this tool has a GUI and should depend on X11, I'm not sure how best to do this. Existing easyconfigs for IGV don't depend on X11.

### CODE REVIEW

**Comment by nathanhaigh** [comment]
Depends on PR #5672

File: easybuild/easyconfigs/i/IGV/IGV-2.4.6-Java-1.8.0_152.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+source_urls = ['http://data.broadinstitute.org/igv/projects/downloads/2.4']
+sources = ['%(name)s_%(version)s.zip']
+
+java = 'Java'
+javaver = '1.8.0_152'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [
+    (java, javaver),
+    ('X11', '20160819', '-foss-2015b'),
```
Comment (boegel) [inline easybuild/easyconfigs/i/IGV/IGV-2.4.6-Java-1.8.0_152.eb]: @nathanhaigh Same remarks as in #5668 w.r.t. `toolchain` and `%(javaver)s`

File: easybuild/easyconfigs/i/IGV/IGV-2.4.6-Java-1.8.0_152.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+]
+
+# add the installation dir to PATH
+modextrapaths = {
+    'PATH': '',
+}
```
Comment (boegel) [inline easybuild/easyconfigs/i/IGV/IGV-2.4.6-Java-1.8.0_152.eb]: OK to make this a single line

File: easybuild/easyconfigs/i/IGV/IGV-2.4.6-Java-1.8.0_152.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+version = '2.4.6'
+
+homepage = 'http://www.broadinstitute.org/software/igv/'
+description = """ The Integrative Genomics Viewer (IGV) is a high-performance visualization 
+ tool for interactive exploration of large, integrated genomic datasets. It supports a wide 
+ variety of data types, including array-based and next-generation sequence data, and genomic annotations. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://data.broadinstitute.org/igv/projects/downloads/2.4']
```
Comment (boegel) [inline easybuild/easyconfigs/i/IGV/IGV-2.4.6-Java-1.8.0_152.eb]: please use `%(version_major_minor)s` rather than hardcoding `2.4`

**Comment by migueldiascosta** [comment]
@nathanhaigh, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**